package/src/ssr/NextSSRApolloClient.tsx

```diff
@@ -1,16 +1,17 @@
-import {
-  ApolloClient,
+import type {
   ApolloClientOptions,
   OperationVariables,
   WatchQueryOptions,
-  Observable,
   FetchResult,
-  DocumentNode,
+  DocumentNode} from "@apollo/client";
+import {
+  ApolloClient,
+  Observable
 } from "@apollo/client";
 import type { QueryManager } from "@apollo/client/core/QueryManager";
-import { print } from "graphql";
```

Comment:
This is what apparently has the big bundle size impact, as for some reason that's not tree-shaken by Next.js otherwise.